pubspec.yaml

```diff
@@ -58,6 +58,7 @@ dependencies:
   connectivity_plus: ^2.2.0
   package_info_plus: ^1.3.0
   flutterfire_installations: ^0.1.0
+  dotted_border: ^2.0.0+2
```

Comment:
@RaulUrtecho  can you check if we need a lib for this? 
And if so, Is this lib good?
Or should we just copy the code in this lib. ?

Reply:
Right we can just copy the implementation this package its 100% Dart code only contains 2 files.
All the code consist in jus a Stack and a custom painter which does the trick.

<img width="331" alt="image" src="https://user-images.githubusercontent.com/42857405/155924513-23e7207a-7a8e-42de-aa2a-b53ec76b95fa.png">

Reply:
@gguij004 we can go even further and remove almost all those unnecessary custom params because we only use a dash, size, colors, etc. for our porpuses.